shared/assets/svgs/backArrow.js

```diff
@@ -0,0 +1,5 @@
+export default `
```

Comment:
I don;t think these are actually being used. because the consumer js files are using relative imports